openpype/plugins/publish/extract_burnin.py

```diff
@@ -156,6 +155,11 @@ def main_process(self, instance):
             filled_anatomy = anatomy.format_all(burnin_data)
             burnin_data["anatomy"] = filled_anatomy.get_solved()
 
+            # Add context data burnin_data.
+            burnin_data["context"] = (
+                instance.context.data.get("burnin_context") or {}
+            )
```

Comment:
I actually have one more idea here after a chat in the office. How about we collect it from instance directly and make it completely optional to whatever the developer needs. that way you could pass in both instance and context keys. Up to you whether you make `custom[context][some_key]` or `custom[instance][some_otherKey]` or `custom[my_key]`.


```suggestion
            # Add context data burnin_data.
            burnin_data["custom"] = (
                instance.data.get("custom_burnin_data") or {}
            )
```

Reply:
The only issue now is that you'll need to get the instance, instead of being able to collect context data at any point during the collection stage.